eng/templates/Azure.ServiceTemplate.Template/content/tests.yml

```diff
@@ -0,0 +1,7 @@
+trigger: none
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
```suggestion
  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Reply:
same as above

Reply:
It would still be good to change this path to be absolute.

Reply:
updated